opacus/utils/batch_memory_manager.py

```diff
@@ -23,7 +23,6 @@
 )
 from torch.utils.data import BatchSampler, DataLoader, Sampler
 
-
```

Comment:
Could you add this missing line back? The current version fails the lint test (https://app.circleci.com/pipelines/github/pytorch/opacus/3256/workflows/26b31f87-1e6e-4562-b070-59dc983bdf4e/jobs/18514?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary). Then I will approve the pr. Thx!